german/GUI_Automation/src/test/java/at05ui/steps/SendStep.java

```diff
@@ -0,0 +1,48 @@
+package at05ui.steps;
+
+import at05ui.sampleapp.Entities.Mail;
+import at05ui.sampleapp.ui.InboxPage;
+import at05ui.sampleapp.ui.LoginPage;
+import at05ui.sampleapp.ui.NextPage;
+import at05ui.sampleapp.ui.PageTransporter;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.When;
+
+/**
+ * Created by Administrator on 11/20/2017.
+ */
+public class SendStep {
+  LoginPage loginPage;
+  InboxPage inboxPage;
+  NextPage nextPage;
+  Mail mail;
+
+  public SendStep(Mail mail) {
+    this.mail = mail;
+    this.mail.setBody("this is a mail");
+    //this.mail.setSubject("AT05GUI");
+    this.mail.setTo("german.eduardo.qm@gmail.com");
```

Comment:
You can get this email direction from environments.json file or as environment variable

Reply:
I will do so